crates/ark/src/ui/events.rs

```diff
@@ -25,9 +25,12 @@ pub unsafe extern "C" fn ps_ui_show_message(message: SEXP) -> anyhow::Result<SEX
         message: RObject::view(message).try_into()?,
     };
 
-    let main = RMain::get();
     let event = UiFrontendEvent::ShowMessage(params);
-    main.send_frontend_event(event);
+
+    let main = RMain::get();
+    let ui_comm_tx = main.get_ui_comm_tx().ok_or_else(ui_comm_not_connected)?;
+    ui_comm_tx.send_event(event);
```

Comment:
Previously would eventually just do

```
log::info!("Discarding message {msg:?}; no frontend UI comm connected");
```

but i think we should actually error here